### ISSUE TITLE
"Unknown filesystem type on /dev/mapper/docker-202" when trying to build image.

### ISSUE THREAD

**Issue by Daenyth** [docker/docker#16552]
I'm trying to build a new image from one of my base images, and when build runs, it failed with something like this:

```
Step 1 : RUN npm install -g gulp karma-cli mocha coffee-script
Unknown filesystem type on /dev/mapper/docker-202:16-802820-5a81f86e959d602cfe6b7de45f497e59e05b80f0b8817eb780842dea7dd7fabc-init
```

I'm on Ubuntu 12.04.

```
$ docker version
Client version: 1.7.1
Client API version: 1.19
Go version (client): go1.4.2
Git commit (client): 786b29d
OS/Arch (client): linux/amd64
Server version: 1.7.1
Server API version: 1.19
Go version (server): go1.4.2
Git commit (server): 786b29d
OS/Arch (server): linux/amd64
```

I have my `/etc/default/docker` configured to pass `-g /mnt/docker`.

```
$ mount | grep '/mnt'
/dev/xvdb on /mnt type ext3 (rw,_netdev)
```

`/dev/xvdb/` is backed by an Amazon EBS.

**Comment by Daenyth** [comment]
`docker info`
```
Containers: 150
Images: 105
Storage Driver: devicemapper
 Pool Name: docker-202:16-802820-pool
 Pool Blocksize: 65.54 kB
 Backing Filesystem: extfs
 Data file: /dev/loop2
 Metadata file: /dev/loop3
 Data Space Used: 4.579 GB
 Data Space Total: 107.4 GB
 Data Space Available: 41.57 GB
 Metadata Space Used: 7.287 MB
 Metadata Space Total: 2.147 GB
 Metadata Space Available: 2.14 GB
 Udev Sync Supported: false
 Deferred Removal Enabled: false
 Data loop file: /mnt/docker/devicemapper/devicemapper/data
 Metadata loop file: /mnt/docker/devicemapper/devicemapper/metadata
 Library Version: 1.02.82-git (2013-10-04)
Execution Driver: native-0.2
Logging Driver: json-file
Kernel Version: 3.13.0-40-generic
Operating System: Ubuntu 14.04.3 LTS
CPUs: 4
Total Memory: 7.337 GiB
Name: ip-10-146-205-21
ID: WS7K:JB3C:5XHN:HGHN:BM7Y:Q3UA:I64G:P26V:MBMC:DDVG:AYYB:OLMU
WARNING: No swap limit support
```

`uname -a`
```
Linux ip-10-146-205-21 3.13.0-40-generic #69-Ubuntu SMP Thu Nov 13 17:53:56 UTC 2014 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by LK4D4** [comment]
Wasn't reported since, code changed a lot. Feel free to open a new issue if it's still a problem,
